### PR TITLE
UpdateExtent and Call ViewPort Changes on Transform

### PR DESCRIPTION
**Pull request by inforithmics**
Maybe this fixes this.
https://github.com/Mapsui/Mapsui/issues/1830

### CODE REVIEW

File: Mapsui/Viewport.cs [hunk 1]
```diff
@@ -247,13 +247,16 @@ public MPoint WorldToScreen(double worldX, double worldY)
     public void Transform(MPoint positionScreen, MPoint previousPositionScreen, double deltaResolution = 1, double deltaRotation = 0)
     {
         _animations = new();
-
+        bool changes = false;
```
Comment (pauldendulk) [inline Mapsui/Viewport.cs:250]: It looks like this field is not used anywhere.

**Review by pauldendulk — APPROVED** [review]

**Comment by janusw** [comment]
Thanks for this PR! I'll try to check ASAP if it actually fixes #1830 for me ...

**Comment by janusw** [comment]
In fact this PR seems to fix #1830 AFAICT (I have tested version 4.0.0-beta.7-89-g640c84e1c, corresponding to the merge commit). Thanks a lot for the quick fix, @inforithmics!

**Comment by pauldendulk** [comment]
🎉Great testing, great fixing, everything is wonderful, lets have a party! I going to release and then enjoy the weekend.

**Pull request merged**